config/requests.yml.tmpl

```diff
@@ -0,0 +1,14 @@
+# requests.yml
+
+defaults: &defaults
+  umlaut_base: https://getit.princeton.edu
+  bibdata_base: http://bibdata.princeton.edu/bibliographic
```

Comment:
If we wind up adding a lot of config that needs to be exposed to JS we could create some sort of config object that gets registered in the Blacklight JS namespace. For a few values we could just use data attributes that our custom JS knows to pull when it needs something like this.